pkg/controllers/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -94,11 +93,12 @@ func NewConfigObserver(
 		PreRunCachesSynced:   preRunCacheSynced,
 	}
 
-	// Check if the Console capability is enabled on the cluster and sync and add its informer and lister.
+	// Check if the Console capability is enabled on the cluster and sync and add its informer, lister, and config observer
 	if enabledClusterCapabilities.Has("Console") {
 		listers.PreRunCachesSynced = append(listers.PreRunCachesSynced, configInformer.Config().V1().Consoles().Informer().HasSynced)
 		informers = append(informers, configInformer.Config().V1().Consoles().Informer())
 		listers.ConsoleLister = configInformer.Config().V1().Consoles().Lister()
+		oauthServerObservers = append(oauthServerObservers, configobserver.WithPrefix(console.ObserveConsoleURL, configobservation.OAuthServerConfigPrefix))
```

Comment:
this wasn't strictly needed but there's no reason to run this observer as long as the console isn't enabled.  (the observer itself no-ops if the console cap is disabled)